.github/workflows/dockerimage.yml

```diff
@@ -16,7 +19,7 @@ jobs:
     steps:
     - name: 'Checkout Github Action'
       uses: actions/checkout@master
-      
+
```

Comment:
It's not common to checkout the same repo multiple times, we can add comments here to explain why we are using this somehow weired workaround.

Reply:
I remove the second code checkout now, and this workflow is triggered only when a new release is published.